06_neighborhoods/Transportation/transit_trips_county.R

```diff
@@ -0,0 +1,201 @@
+###################################################################
+
+# Transit Metrics: Transit Trips
+# Tina Chelidze 
+# 2023-2024 Mobility Metrics update
+# Process (for overall, not subgroup, data):
+# 1. For County: Import and reshape data
+# 2. QC checks
+# 3. Add Data Quality marker
+# 4. Export final files
+
+###################################################################
+
+
+# Libraries you'll need
+library(tidyr)
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(tidyverse)
+library(purrr)
```

Comment:
Tidycensus is still missing from the list of packages

Reply:
Added